File: server-mh/src/Match/Handler/PieceState.ts
export class PieceState {
    public spawned: boolean;
    public inHome: boolean;
    public finished: boolean;
    public hasLeftStart: boolean;

    constructor(
        spawned: boolean = false,
        inHome: boolean = true,
        finished: boolean = false,
        hasLeftStart:boolean=false
    ) {
        this.spawned = spawned;
        this.inHome = inHome;
        this.finished = finished;
        this.hasLeftStart=hasLeftStart;
    }
}

File: Client-unity/Ludo/Assets/Scripts/OfflineMatchHandler/PieceState.cs
namespace Ludo.Offline
{
    public class PieceState
    {
        public bool Spawned;
        public bool Finished;
        public bool HasLeftStart;

        public PieceState()
        {
            Spawned = false;
            Finished = false;
            HasLeftStart = false;
        }
    }
}